lib/katello/permissions/host_permissions.rb

```diff
@@ -56,6 +56,7 @@
   'katello/api/v2/hosts_bulk_actions/module_streams',
   'katello/api/v2/host_debs/index',
   'katello/api/v2/host_packages/index',
+  'katello/api/v2/host_packages/containerfile_install_command',
```

Comment:
This was missing before. I never tested other users, only admin. It was caught via unit test. I believe without it, the endpoint still works for admins, but never works for other users. Now it should work for users that actually have the right permissions.